src/js/edu-benefits/5490/components/ServicePeriodView.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
If this is the same as the 1990, we'll probably want to share it. So maybe remove it for now.